controllers/bookController.js

```diff
@@ -0,0 +1,19 @@
+const bookService = require('../services/bookService.js');
+const { catchAsync } = require('../middlewares/error.js');
+
+const getBookById = catchAsync(async (req, res) => {
+  const { bookId } = req.params;
+  if (!bookId) {
```

Comment:
1. if 시작 할 때 공백 추가해 주세요.
```suggestion
  const { bookId } = req.params;
  
  if (!bookId) {
```